src/resource_dasm.cc

```diff
@@ -494,6 +494,32 @@ class ResourceExporter {
       }
       this->write_decoded_data(base_filename, res, ".bmp", img);
     }
+    
+    // Also store the colors as a Photoshop .act file
+    {
+      // https://www.adobe.com/devnet-apps/photoshop/fileformatashtml/#50577411_pgfId-1070626
+      //
+      //    There is no version number written in the file. The file is 768 or 772
+      //    bytes long and contains 256 RGB colors. The first color in the table
+      //    is index zero. There are three bytes per color in the order red, green,
+      //    blue. If the file is 772 bytes long there are 4 additional bytes remaining.
+      //    Two bytes for the number of colors to use. Two bytes for the color index
+      //    with the transparency color to use. If loaded into the Colors palette,
+      //    the colors will be installed in the color swatch list as RGB colors.
+      //
+      StringWriter  data;
+      for (size_t z = 0; z < decoded.size(); z++) {
+        data.put_u8(decoded[z].c.r / 0x0101);
+        data.put_u8(decoded[z].c.g / 0x0101);
+        data.put_u8(decoded[z].c.b / 0x0101);

```

Comment:
`decoded[z].c.as8()` will give you an object that has 8-bit channels, which can be sent directly to `data.put_u8`. This is fine too though.